src/data/github-projects.json

```diff
@@ -1,5 +1,25 @@
 {
   "projects": [
+    {
+      "title": "GitLab",
+      "description": "GitLab is a complete open-source DevOps platform, delivered as a single application, fundamentally changing the way Development, Security, and Ops teams collaborate and build software.",
+      "language": "Javascript",
+      "tags": [
+        {
+          "src": "https://img.shields.io/static/v1.svg?label=Contributions&message=Welcome&color=0059b3&style=flat-square",
+          "href": "https://about.gitlab.com/community/contribute/",
+          "alt": "Contributions welcome"
+        },
+        {
+          "src": "https://img.shields.io/badge/Chat-Gitter-ff69b4.svg?label=Chat&logo=gitter&style=flat-square",
+          "href": "https://gitter.im/gitlab/contributors",
+          "alt": "Gitter chat"
+        }
+      ],
+      "repoName": "gitlabhq/gitlabhq",
+      "link": "https://gitlab.com/gitlab-org/gitlab",
+      "logo": "gitlab.png"
+    },
```

Comment:
Please move this below Tesseract.js i.e below this line https://github.com/gitpod-io/contribute.dev/blob/c642b014b72a06b5a74ab01dcd34f2a6d55d4843/src/data/github-projects.json#L210 because the projects are sorted in descending order based on the number of stars.

![image](https://user-images.githubusercontent.com/46004116/99771779-7d420980-2b2b-11eb-8a33-a455bfe7fc29.png)

Sorry for the in-convienience We have the `yarn add-project` script which automates most of the stuff i.e sorting and so on and jus requires the `repoURL` as an argument but currently it does not work and needs some fixes and updates.

Reply:
Sorry you can ignore this we have decided thta Gitlab can be the very first project in the list.